app/views/users/cards/_submission.html.erb

```diff
@@ -5,10 +5,10 @@
   </td>
   <td>
     <h4 class='text-<%= submission.class.to_s.downcase %>'>
-      <i class='fa <%= submission.icon %>'></i> 
-      <%= link_to submission.title, after_update_object_path_for(submission) %>
+      <i class='fa <%= submission.icon %>'></i>
+      <%= link_to submission.title.html_safe, after_update_object_path_for(submission) %>
     </h4>
-    <p class='zero-margin-bottom'><%= submission.description %></p>
+    <p class='zero-margin-bottom'><%= submission.description.html_safe if submission.description.present? %></p>
```

Comment:
I think we always require description with every submission, so we can avoid the extra check here for `if submission.description.present?` – know that I suggested it, but think we'll be fine without it.